tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts

```diff
@@ -41,7 +41,7 @@ export default {
                 "Litmus",
                 "LitentryRococo",
                 "Khala",
-                "SubstrateTestnet",
+                __Unused6: "Null",
```

Comment:
The syntax is wrong tho 😂 

Btw is `Web3Network` actually used? Shouldn't we use `CorePrimitivesNetworkWeb3Network` ubiquitously? @Verin1005 I thought this problem was fixed already

Reply:
I checked, and we are still using the `Web3Network` at the moment.

Reply:
Hmm why is that? I thought it had been mentioned multiple times that we'd need to use auto-generated types :(

Reply:
We can indeed replace the introduced [type](https://github.com/litentry/litentry-parachain/blob/p-92-remove-substratetestnet-from-web3network/tee-worker/ts-tests/integration-tests/di_evm_identity.test.ts#L28) with `CorePrimitivesNetworkWeb3Network`.

But this PR update will have an impact on these [definitions](https://github.com/litentry/litentry-parachain/blob/p-92-remove-substratetestnet-from-web3network/tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts#L36C105-L36C116) , so `Web3Network` actually used.

Reply:
Thanks for checking, why can't we use `CorePrimitivesNetworkWeb3Network` in the definitions? Or is there a way to do that?

Reply:
I have made updates [here](https://litentry.slack.com/archives/C05328L0GAF/p1703884176997769), and the conclusion is negative. @jonalvarezz 's approach is not feasible in our case; we cannot manually import types every time.

Reply:
Hmm so we can't define the source like types from registry or lookup?

Reply:
As of now, it seems not possible; it cannot be automatically imported. I've also referenced Moonbeam repo and haven't seen them do something similar at the moment. :(